scss/style.scss

```diff
@@ -0,0 +1,1292 @@
+@font-face {
+    font-family: OpenSans;
+    src: url(font/OpenSans/OpenSans-Bold.ttf);
+}
+
+body {
+    margin: 0px;
+    width: 100%;
+}
+
+.container {
+    margin-left: 10%;
+    margin-right: 10%;
+    width: 80%;
+}
+
+.header {
+
+    &__top {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        width: 100%;
+        height: 100px;     
+
+        &__logo {
+            font-family: OpenSans;
+            font-size: 25px;
+            color: #212121;
```

Comment:
You need to add variables with colors which are used more than 2 times in the code!